src/cobalt_model/permalink.rs

```diff
@@ -24,7 +24,7 @@ fn explode_permalink_string(permalink: &str, attributes: &liquid::value::Object)
     // Handle cases where substutions were blank
     p = p.replace("//", "/");
 
-    if p.starts_with('/') {
+    if p.starts_with('/') && p.len() > 1 {
```

Comment:
I've added this to avoid empty `doc.url_path` which broke some pagination navigation

Reply:
Any idea why?  If so, we should have test cases to clarify..

Reply:
I'll revert this, I think it's the same issue of the leading slash to be added by the user